### PR TITLE
Fix/deployment goerli

### PR DESCRIPTION
**Pull request by mathewmeconry**
## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

Task: [ID]()

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: packages/contracts/Releases.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Contract Releases
 
+Commit: [f8aee0a0fa7ae5825130b9726a18ef6e36ddc121](https://github.com/aragon/core/commit/f8aee0a0fa7ae5825130b9726a18ef6e36ddc121)
```
Comment (brickpop) [inline packages/contracts/Releases.md:3]: Can we update the script that generates this?  `core` should now be `osx` in the URL

File: packages/contracts/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aragon/core-contracts",
-  "version": "0.7.1-alpha",
+  "version": "0.8.0-alpha",
```
Comment (brickpop) [inline packages/contracts/package.json:3]:
```suggestion
  "version": "0.8.0-beta",
```

File: packages/contracts-ethers/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aragon/core-contracts-ethers",
-  "version": "0.7.1-alpha",
+  "version": "0.8.0-alpha",
```
Comment (brickpop) [inline packages/contracts-ethers/package.json:3]:
```suggestion
  "version": "0.8.0-beta",
```

File: packages/subgraph/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aragon/core-subgraph",
-  "version": "0.9.0-alpha",
+  "version": "0.10.0-alpha",
```
Comment (brickpop) [inline packages/subgraph/package.json:3]: Are we publishing subgraph via NPM?

```suggestion
  "version": "0.10.0-beta",
```

File: packages/contracts/Releases.md (hunk 1 above)
Comment (mathewmeconry) [reply packages/contracts/Releases.md]: done

**Review by brickpop — DISMISSED** [review]

**Review by novaknole — DISMISSED** [review]

File: packages/contracts/networks.json [hunk 5]
```diff
@@ -2,7 +2,8 @@
   "goerli": {
     "url": "https://goerli.infura.io/v3/92aa62d2bb5449cfafe04b83ca8636f1",
     "isTestnet": true,
-    "chainId": 5
+    "chainId": 5,
+    "gasPrice": 100000000000
```
Comment (Rekard0) [inline packages/contracts/networks.json:6]: should we leave this here?
perhaps we can grab it from .env and add in hardhat.config.ts
just a suggestion, feel free to resolve this comment

**Review by Rekard0 — DISMISSED** [review]

File: packages/contracts/networks.json (hunk 5 above)
Comment (mathewmeconry) [reply packages/contracts/networks.json]: good finding. removed

**Pull request merged**